ReadsUtils.spec

```diff
@@ -55,10 +55,20 @@ module ReadsUtils {
         
         If local files are specified for upload, they must be uncompressed.
         Files will be gzipped prior to upload.
+
+        If web files are specified for upload, a download type one of
+        ['Direct Download', 'DropBox', 'FTP', 'Google Drive'] must be specified too. 
+        The downloadable file must be uncompressed (except for FTP, .gz file is acceptable). 
```

Comment:
I put a decompression for just FTP because it's just a temp solution for Chris and Sunita's test. Ideally, after contractor's demo, I can do a concrete decompression in FDU.